shared/fs/row/index.js

```diff
@@ -1,7 +1,11 @@
 // @flow
 import Placeholder from './placeholder'
+import TlfType from './tlf-type-container'
+import Tlf from './tlf-container'
```

Comment:
Instead of doing all these imports then a single export, you can do:
```javascript
export {default as Tlf} from './tlf-container'
```

Reply:
I also believe having separate row types for `TlfType` and Tlf` is overkill - it forces you to do further indirection like `StillCommon` and `StillCommonProps`, making it painful to find the one place you finally need to implement a change. `TlfType` and `Tlf` should just exist in the same place as `Still`, and all the encapsulation should occur in that file. Callers should just include a `Still` row as necessary.